src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -199,6 +206,26 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
     }
   }
 
+  private def commitPurgeSeq(newSeq: Long) {
```

Comment:
You mean we can commit `purgeSeq` and `updateSeq` with `commit` function?
We can change `commit` function with 2 arguments to support it.
```
  private def commit(newSeq: Long, type: String) {
    if (type == "updateSeq") {
      ...
    } else {
      ...
    }
```